.github/workflows/protobuf.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: Protobuf specs
+on: [pull_request]
+jobs:
+  protobuf-fresh:
```

Comment:
We already have a file defining GH Actions for protobufs -- could we put this action in the same file as those, so we have one place for all of the proto-related actions?